CRM/Civicase/Hook/BuildForm/HandleDraftActivities.php

```diff
@@ -13,7 +13,7 @@ class CRM_Civicase_Hook_BuildForm_HandleDraftActivities {
   ];
 
   const PDF_LETTER_FORM_NAME = 'CRM_Contact_Form_Task_PDF';
-  const EMAIL_FORM_NAME = 'CRM_Contact_Form_Task_Email';
+  const EMAIL_FORM_NAME = 'CRM_Case_Form_Task_Email';
```

Comment:
I test the functionality of create pdf letter and it is working fine, and I think there is no problem because in https://github.com/civicrm/civicrm-core/blob/edc52241c6be939474660e67a968194b9177537b/CRM/Case/Form/CaseView.php#L259-L263 (in the core) they still uses `civicrm/activity/pdf/add` which uses `CRM_Contact_Form_Task_PDF`

Reply:
ah okay, fine then.